src/dmd/mtype.d

```diff
@@ -1038,6 +1039,11 @@ extern (C++) abstract class Type : ASTNode
         return false;
     }
 
+    bool ischar()
```

Comment:
This virtual function goes around the horn doing something trivial, i.e.:
```
final bool isChar() { return ty == Tchar || ty == Twchar || ty == Tdchar; }
```
done! Delete the rest of the changes in this file.